2Dgame/Assets/Scripts/Platform Scripts/BlinkerScript.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BlinkerScript : PlatformScript {
+
+    new Renderer renderer;
+
+    private new void Awake()
+    {
+        renderer = GetComponent<Renderer>();
+    }
+
+    /**
+     * Enables the renderer when jumped on
```

Comment:
Don't seem to have this problem, you probably don't have the latest commits on this branch